frontend/public/components/build-config.tsx

```diff
@@ -219,6 +243,13 @@ const isBuildNewerThen = (newBuild: K8sResourceKind, prevBuild: K8sResourceKind
 const buildStrategy = (buildConfig: K8sResourceKind): BuildStrategyType =>
   buildConfig.spec.strategy.type;
 
+const buildStatus = (buildConfig: BuildConfig) => {
+  if (buildConfig.latestBuild) {
+    return buildConfig.latestBuild?.status?.phase;
+  }
+  return 'Unknown';
```

Comment:
Shouldn't this return 'Unknown' also when there is no status or no phase?

```suggestion
const getBuildStatus = (buildConfig: BuildConfig) => {
  return buildConfig?.latestBuild?.status?.phase || 'Unknown';
```